### DISCUSSION TITLE
Pass computed value for livewire

### DISCUSSION THREAD

**Discussion by NetRally** [alpinejs/alpine#1334] [1. Help]
Hello!
I hope somebody can help. I try to pass from Alpine computed value for livewire and the input field
  :value="JSON.stringify(tags)"
won't send the data.
 
 :value="$wire.set('tags', JSON.stringify(tags))"
pass the variable and livewire gets it, but $wire.set always send never ending query for the server in background.
Can somebody suggest an elegant solution to send the data and won't generate unnecessary trafic?

**Comment by gwleuverink** [comment]
It is not exactly clear te me what you are trying to do. Maybe look into `@entangle`?

**Reply by NetRally** [reply]
Meanwhile I find the solution and is not in documentation
$wire.set need third parameter true to skip watch function and it won't send data in the background if no need
$wire.set('param','value',true)

**Reply by gwleuverink** [reply]
Glad you found a workaround. I noticed you use $wire.set inside of a bound value attribute which makes me wonder if there is a more elegant solution to your problem.

If you'd like to get to the bottom of this please provide me with some more context on the problem you want to solve and I'll try to help you out
